scss/civicrm/common/_bootstrap-calendar.scss

```diff
@@ -1,5 +1,4 @@
-.page-civicrm {
-  .ui-datepicker {
+.ui-datepicker {
     z-index: 9999 !important;
```

Comment:
I think there are 4 spaces, should be 2? Please fix the indentation in the whole file.

Reply:
Fixed!